test/convert.js

```diff
@@ -0,0 +1,21 @@
+'use strict'
+
+const tape = require('tape')
+const { BufferList, BufferListStream } = require('../')
+const { Buffer } = require('safe-buffer')
+
+tape('convert from BufferList to BufferListStream', t => {
+  const data = Buffer.from(`TEST-${Date.now()}`)
+  const bl = new BufferList(data)
+  const bls = new BufferListStream(bl)
+  t.ok(bl.slice().equals(bls.slice()))
```

Comment:
These two were `t.equal(bl.slice(), bls.slice())` in #73, strict equality rather than buffer equality. These don't work because #72 unwraps the `Buffer` instances and rebuilds a new one from the underlying `ArrayBuffer`. Does this matter to anyone?

Reply:
Considering the documentation has always said that `slice` returns a **new** `Buffer` I don't think there should be any expectations that it returns the _same_ buffer. I think the test was wrong.